iamy/models.go

```diff
@@ -145,6 +154,7 @@ type AccountData struct {
 	Roles          []*Role
 	Policies       []*Policy
 	BucketPolicies []*BucketPolicy
+	InstanceProfiles []*InstanceProfile
```

Comment:
we should look to align all of these struct fields

Reply:
043328d